v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,131 @@
+
+Revising our sample scenario, suppose we want to create a new user on the social networking app. We need to create entries for the last name of the user, their favorite movie, and favorite song. We need to insert entries into three tables: users, favorite_movies, and favorite_songs. The transaction would be as follows:
+
+~~~ sql
+> BEGIN;
+> INSERT INTO users VALUES last_name = 'Pavlo' WHERE id = 2 RETURNING NOTHING;
+> INSERT INTO favorite_movies VALUES movies = 'Godfather' WHERE user_id = 2 RETURNING NOTHING;
+> INSERT INTO facvorite_songs VALUES songs = 'Remember' WHERE user_id = 2 RETURNING NOTHING;
+> COMMIT;
+~~~
+
+In this case, the second and third `INSERT` statements are dependent on the first `INSERT` statement because the movies and songs tables both have a foreign key constraint on the users table. So even though we append the `RETURNING NOTHING` clause to the first statement, CockroachDB executes the first statement sequentially. After the first statement is executed to completion, the second and third `INSERT` statements are executed in parallel. The following conceptual diagram shows how the transaction is executed in sequential and parallel modes:
+
+<img src="{{ 'images/Parallel_Statement_Hybrid_Execution.png' | relative_url }}" alt="CockroachDB Parallel Statement Hybrid Execution" style="border:1px solid #eee;max-width:100%" />
+
+## Independent SQL statements
+
+As mentioned earlier, CockroachDB considers SQL statements within a single transaction to be independent if their execution can be safely reordered without affecting their results. 
+
+For example, the following statements are considered independent since reordering the statements does not affect the results:
+
+~~~ sql
+> INSERT INTO a VALUES (100);
+> INSERT INTO b VALUES (100);
+~~~
+
+~~~ sql
+> INSERT INTO a VALUES (100);
+> INSERT INTO a VALUES (200);
+~~~
+
+The following pairs of statements are dependent since reordering them will affect their results:
+
+~~~ sql
+> UPDATE a SET b = 2 WHERE y = 1;
+> UPDATE a SET b = 3 WHERE y = 1;
+~~~
+
+~~~ sql
+> UPDATE a SET y = true  WHERE y = false;
+> UPDATE a SET y = false WHERE y = true;
+~~~
+
+
+{{site.data.alerts.callout_info}}Parallel statement execution in CockroachDB is different than parallel query execution in PostgreSQL. For PostgreSQL, parallel query execution refers to “creating multiple query processes that divide the workload of a single SQL statement and executing them in parallel”. For CockroachDB’s parallel statement execution, an individual SQL statement is not divided into processes. Instead, multiple independent SQL statements within a single transaction are executed in parallel.{{site.data.alerts.end}}
+
+## See Also
```

Comment:
Let's also link to `[Transactions](transactions.html)`.

Reply:
Fixed.